frontends/web/src/routes/account/send/send.tsx

```diff
@@ -65,24 +125,24 @@ export default class Send extends Component {
             signConfirm: null, // show visual BitBox in dialog when instructed to sign.
             coinControl: false,
             activeCoinControl: false,
+            data: null,
         };
-        this.selectedUTXOs = [];
     }
 
-    coinSupportsCoinControl = () => {
-        const account = this.getAccount();
+    private coinSupportsCoinControl = () => {
+        const { account } = this.state;
```

Comment:
Is this preferable to `const account = this.state.account;`? And since you use the account twice at most, I feel that the local const is no longer really necessary.